### PR TITLE
Create AnaColon_resumen_ Unomia stolonifera.md

### PR DESCRIPTION
**Pull request by AnaColon**
This is the summary for the Unomia stolonifera Cuba paper that was proposed for our second Spanish prompt.

### CODE REVIEW

File: prompts/2023/español/idea_2023_02/AnaColon_resumen_ Unomia stolonifera.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Interesting thing to note: instead of writing coral reefs as arrecifes de coral, this paper wrote arrecifes coralinos.
```
Comment (rmcminds) [inline prompts/2023/español/idea_2023_02/AnaColon_resumen_ Unomia stolonifera.md:1]: Does this have a slightly different connotation to you?

File: prompts/2023/español/idea_2023_02/AnaColon_resumen_ Unomia stolonifera.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+In September of 2022 the ecological evaluation project labelled “Habana sumergida” filmed an image of a soft coral (Octocorallia). This was not a known species for the Cuban specialists until this moment. They consulted what is called “Red de Blanqueamineto” which translates to the “Whitening Network”; and I am assuming based on context that it means coral reef bleaching. This lead to it being believed that the soft coral could be *Unomia stolonifera* which is an invasive species of coral that was first seen in the Caribbean on the coasts of Venezuela in 2014. 
+
+It was found that the morphology of the colony, the polyp pulsing, and the pinnules arrangement were those of *U. stolonifera*. However, it was noted that for a more accurate classification, a DNA study is needed. 
+
+The coral was confined to one section of the reef (23° 10’ 47.4" N and 82° 14’ 39.7"). The coral was extracted from the seafloor with the permission of the Oficina de Regulación y Seguridad Ambiental (ORSA) (Office of Environmental Regulation and Safety).
```
Comment (rmcminds) [inline prompts/2023/español/idea_2023_02/AnaColon_resumen_ Unomia stolonifera.md:7]: Spanish and French use commas and periods the opposite way we use them in English, so the GPS coordinates needed translation, too! (This is actually important to remember - these kinds of formatting differences often make their way into datasets and can really mess up data analysis if improperly parsed)

**Review by rmcminds — CHANGES_REQUESTED** [review]
I'm doing this review with the phone app, and it didn't let me create a new branch for my suggested changes. Feel free to manually changes something back if you'd like.

For thoroughness, I'll suggest that you replace the spaces in the filename with underscores before I fully approve the summary. It's good to get used to - spaces in filenames just cause issues down the line whenever you want to use code.

File: prompts/2023/español/idea_2023_02/AnaColon_resumen_ Unomia stolonifera.md (hunk 1 above)
Comment (AnaColon) [reply prompts/2023/español/idea_2023_02/AnaColon_resumen_ Unomia stolonifera.md:1]: No, I just wanted to note the differences we might see when referring to coral reefs in texts.

**Review by rmcminds — APPROVED** [review]

**Pull request merged**